pkg/components/node-problem-detector/component.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package nodeproblemdetector is code for node-problem-detector.
+package nodeproblemdetector
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+
+	"github.com/kinvolk/lokomotive/internal/template"
+	"github.com/kinvolk/lokomotive/pkg/components"
+	"github.com/kinvolk/lokomotive/pkg/components/util"
+	"github.com/kinvolk/lokomotive/pkg/k8sutil"
+)
+
+const (
+	// Name represents node-problem-detector component name as it should be referenced in function calls
+	// and in configuration.
+	Name = "node-problem-detector"
+)
+
+type component struct {
+	Namespace         string
```

Comment:
If we have no `hcl` tag, is it user configurable? If not do we need it?

Also, I'd suggest sort the fields to have HCL on top.

Reply:
It's not user-configurable. The chart has to be installed in k`ube-system` only. Also we do need it

Reply:
Why do we need it then?

Reply:
Oh, got it what you mean